packages/server-wallet/e2e-test/pong/controller.ts

```diff
@@ -17,29 +46,28 @@ export default class PongController {
     // types of messages between parties. e2e-test uses wire-format
     const convertedSignedStates = (signedStates as unknown) as SignedState[];
 
-    const {channelResults, outbox} = await this.wallet.pushMessage({
+    const {
+      channelResults: [pingsUpdate],
+    } = await this.wallet.pushMessage({
       signedStates: convertedSignedStates,
       to,
       from,
     });
 
-    if (!channelResults) throw Error('sanity check');
+    // FIXME: At this point, pingsUpdate is undefined, but should be a result
+    // waiting on https://github.com/statechannels/statechannels/pull/2370
```

Comment:
Can you call `this.wallet.getChannel` to get around that for now? And maybe `logger.warn("pingsUpdate is undefined")` or leave a `TODO` that you shouldn't do that?